### DISCUSSION TITLE
Mysql 5.6 and '0000-00-00 00:00:00' timestamp values give "Value out of range for the type. The column <column> contained an invalid datetime value with either day or month set to zero."

### DISCUSSION THREAD

**Discussion by slvrtrn** [prisma/prisma#8396] [Q&A] [answered]
Hello!

I am working with a MySQL dump that sometimes contains records with '0000-00-00 00:00:00' as timestamp column values. 
When I try to select such a record, I get:
```
Value out of range for the type. The column <column> contained an invalid datetime value with either day or month set to zero.
```
and the query fails.

Unfortunately, it is not possible to modify the database.

What are the other options? 

Cheers.

**Comment by ryands17** [comment, marked as answer]
@slvrtrn 👋 

This is a known issue. I would suggest watching this issue for more updates 🙂 
https://github.com/prisma/prisma/issues/5006

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
